web/client/utils/mapinfo/__tests__/wms-test.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2022, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import expect from 'expect';
+
+import axios from '../../../libs/ajax';
+import MockAdapter from "axios-mock-adapter";
```

Comment:
when you can try to keep external libraries separated by project modules and to be listed as the first thing
```suggestion
import expect from 'expect';
import MockAdapter from "axios-mock-adapter";

import axios from '../../../libs/ajax';
```